.circleci/docker/common/install_gcc.sh

```diff
@@ -7,7 +7,11 @@ if [ -n "$GCC_VERSION" ]; then
   # Need the official toolchain repo to get alternate packages
   add-apt-repository ppa:ubuntu-toolchain-r/test
   apt-get update
-  apt-get install -y g++-$GCC_VERSION
+  if [ "$UBUNTU_VERSION" == "16.04" -a "$GCC_VERSION" == "5" ]; then
```

Comment:
Prefer to use a single `=` within shell conditionals.

https://unix.stackexchange.com/a/72042/249966